packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Passing no AppBarTheme returns defaults', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(appBar: AppBar()),
+    ));
+
+    final Material widget = _getAppBarMaterial(tester);
+
+    expect(widget.color, Colors.blue);
+    expect(widget.elevation, equals(4.0));
```

Comment:
Here and elsewhere: what about brightness, iconTheme, and textTheme?

Reply:
Good point, added for brightness and iconTheme. Having some trouble with testing textTheme and will need to spend some more time looking into it.

Reply:
Added tests for textTheme